sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/DocumentResult.java

```diff
@@ -35,11 +39,11 @@ DocumentResult setTextDocumentStatistics(TextDocumentStatistics textDocumentStat
         return this;
     }
 
-    public DocumentError getError() {
+    public Object getError() {
```

Comment:
need to finalize on the error object

Reply:
let's change to DocumentError

Reply:
do you still want this to be changed to DocumentError?

Reply:
I think it should be DocumentError. The output of the error content is DocumentError.

Reply:
Will resolve this in https://github.com/Azure/azure-sdk-for-java/issues/6559